gitgud/skills/user_messages.py

```diff
@@ -177,3 +177,26 @@ def display_entry(index, human_name, code_name, indent):
                 code_name=item.name,
                 indent=indent
             )
+
+
+def amending_message(before_ref, after_ref, show_hashes=True, show_files=True, show_refs=True):  # noqa: E501
+    file_operator = operations.get_operator()
+    display_data = [
+        {"_name": "Before", "_commit": file_operator.repo.commit(before_ref)},
+        {"_name": "After", "_commit": file_operator.repo.commit(after_ref)}
```

Comment:
Here, it might be worth it to do tuples or nested dictionaries. Using underscores is a bit weird here.